pkgs/tools/audio/nanotts/default.nix

```
@@ -0,0 +1,35 @@
+
+{ lib, stdenv, fetchFromGitHub, autoconf, automake, libtool, popt, alsaLib }:
+
+stdenv.mkDerivation {
+  pname = "nano-tts";
+  version = "unstable-2021-2-22";
+
+  src = fetchFromGitHub {
+    repo = "nanotts";
+    owner = "gmn";
+    rev = "d8b91f3d9d524c30f6fe8098ea7a0a638c889cf9";
+    sha256 = "sha256-bFu3U50zc90iQeWkqOsCipkueJUZI3cW5342jjYSnGI=";
+  };
+  nativeBuildInputs = [ autoconf automake ];
+  buildInputs = [ libtool popt alsaLib ];
```

```suggestion
  strictDeps = true;
  nativeBuildInputs = [ autoconf automake libtool ];
  buildInputs = [ popt alsaLib ];
```

libtool is run during the build so it should be in nativeBuildInputs

strictDeps ensures that dependencies are correctly declared (its not the default yet sadly)